net/ser2net/files/etc/init.d/ser2net

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh /etc/rc.common
+START=75
+STOP=10
+CONFIGFILE="/tmp/ser2net.conf"
+
+start() {
+	pidof ser2net | awk '{print $2}' > /tmp/ser2net
+	PID=$(cat /tmp/ser2net)
+	rm /tmp/ser2net
+    if [ ! $PID ]; then
+		update_configfile
+        ser2net -c "$CONFIGFILE"
+
+    else
+        echo "Ser2Net: is already running"
+    fi
+}
+
+stop() {
+	pidof ser2net | sed -e "s/$$//g" > /tmp/ser2net_pid
+	PID=`cat /tmp/ser2net_pid`
+	rm -f /tmp/se2net_pid
+	kill -9 $PID 	
+}
+
+update_configfile() {
+	echo "Ser2Net: Generating "$CONFIGFILE" based on /etc/config/ser2net (in cbi format)."
+	echo "#This file is generated based on /etc/config/ser2net." > "$CONFIGFILE"
+	i=0
+	while [ true ]
+	do
+	        tcpport=`uci get ser2net.@proxy["$i"].tcpport 2> /dev/null`
+	        if [ -z $tcpport ]; then
+	                break
+	        fi
+	        state=`uci get ser2net.@proxy["$i"].state 2> /dev/null`
+	        timeout=`uci get ser2net.@proxy["$i"].timeout 2> /dev/null`
+	        device=`uci get ser2net.@proxy["$i"].device 2> /dev/null`
+	        options=`uci get ser2net.@proxy["$i"].options 2> /dev/null`
```

Comment:
this is pretty gross.  Look at how other packages use ```config_load``` instead.

Reply:
Ok will look into that.

Reply:
I'll review this part when you updated the PR. Please update this PR and do not create a new one.

Reply:
Just one note: I think about _adding_ the dynamically generated part of the configuration file, i.e. using /etc/ser2net.conf as base and then adding the generated lines add the end. This way, the user can manually edit some defaults or additional items which are not (yet) configurable via UCI but also has the comfort of controlling the main aspects via UCI. What do you think about this idea?

Reply:
That would be feasible indeed. But I also think that if people miss something in our current package they should notify it so it gets supported for everyone. Otherwise people will do local changes and do not bother to mention them upstream. I know this is a little bit harsh but I think it's for the best.

Reply:
But how would you know what the devices were?